poetry.lock

```diff
@@ -1538,6 +1554,9 @@ pip-api = [
     {file = "pip-api-0.0.30.tar.gz", hash = "sha256:a05df2c7aa9b7157374bcf4273544201a0c7bae60a9c65bcf84f3959ef3896f3"},
     {file = "pip_api-0.0.30-py3-none-any.whl", hash = "sha256:2a0314bd31522eb9ffe8a99668b0d07fee34ebc537931e7b6483001dbedcbdc9"},
 ]
+<<<<<<< HEAD
```

Comment:
Your committed lock file has merge conflict markers in it. Delete & re-run `poetry lock` to fix this.